packages/create-expo-module/src/createExampleApp.ts

```diff
@@ -103,13 +103,23 @@ function createCommand(
 }
 
 /**
- * Copies files from one directory to another.
+ * Moves files from one directory to another.
  */
 async function moveFiles(fromPath: string, toPath: string): Promise<void> {
-  for (const file of await fs.readdir(fromPath)) {
-    await fs.move(path.join(fromPath, file), path.join(toPath, file), {
-      overwrite: true,
-    });
+  for (const file of await fs.promises.readdir(fromPath)) {
+    // Overwriting the files means we first need to delete the target
+    await fs.promises.rm(path.join(toPath, file), { force: true, recursive: true });
```

Comment:
qq: should we check whether `toPath` directory exists? fs-extra seems do mkdirp for beforehand